{{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/urls.py

```diff
@@ -65,14 +65,13 @@
     url(r'^projects/', include(project_urls)),
     url(r'^ideas/', include(ideas_urls)),
 {% if cookiecutter.add_documents_app == 'y' %}
-    url(r'^text/', include(documents_urls, '{{ cookiecutter.project_app_prefix }}_documents')),
+    url(r'^text/', include((documents_urls, '{{ cookiecutter.project_app_prefix }}_documents'), '{{ cookiecutter.project_app_prefix }}_documents')),
 {% endif %}
 {% if cookiecutter.add_maps_and_mapideas_app == 'y' %}
     url(r'^mapideas/', include(map_ideas_urls)),
 {% endif %}
     url(r'^components/$', contrib_views.ComponentLibraryView.as_view()),
-    url(r'^jsi18n/$', javascript_catalog,
-        js_info_dict, name='javascript-catalog'),
```

Comment:
I believe this change may cause some test failures - see https://github.com/liqd/a4-cookiecutter/issues/92